tools/requires_log_to_webtreemap.rb

```diff
@@ -0,0 +1,49 @@
+if ARGV.length != 2
+  puts "Usage: #{$0} /path/to/requires.log /path/to/webtreemap"
+  puts
+  puts "This tool uses webtreemap, which can be obtained from https://github.com/evmar/webtreemap"
+  exit 1
```

Comment:
@jrafanie There are no tests around the "tools" dir , so does it make sense to remove that from the coverage report?  Looks like this is the first file added to that.